lib/rules/declaration-no-important/index.js

```diff
@@ -28,10 +28,12 @@ const rule = (primary) => {
 				return;
 			}
 
+			const importantMatch = decl.toString().match(/!\s*important/);
+
 			report({
 				message: messages.rejected,
 				node: decl,
-				word: 'important',
+				word: importantMatch ? importantMatch[0] : 'important',
```

Comment:
[note] It may be better to revert the change for `declaration-no-important` and fix it later (via another pull request).

Reply:
I've fixed the `!important` position via 85540e6.